plugins-ui/src/modules/plugin/utils/plugin.util.ts

```diff
@@ -0,0 +1,23 @@
+import { PluginFee } from "../models/plugin";
+
+const pricingMetric = {
+  FIXED: "USDC",
+  PERCENTAGE: "%",
+};
+
+const pricingType = {
+  FREE: "free",
+  SINGLE: "per installation",
+  RECURRING: "per",
+  PER_TX: "per trade",
```

Comment:
"per transaction", since "per trade" may not always be applicable (depending on the plugin type)